src/patternfly/components/Nav/nav.scss

```diff
@@ -550,3 +585,34 @@
   font-weight: var(--pf-c-nav__section-title--FontWeight);
   color: var(--pf-c-nav__section-title--Color);
 }
+
+.pf-c-nav__scroll-button {
+  position: absolute;
+  top: var(--pf-c-nav__scroll-button--Top);
+  bottom: 0;
+  z-index: var(--pf-c-nav__scroll-button--ZIndex);
+  width: var(--pf-c-nav__scroll-button--Width);
+  padding-right: var(--pf-c-nav__scroll-button--PaddingRight);
+  padding-left: var(--pf-c-nav__scroll-button--PaddingLeft);
+  background-color: var(--pf-c-nav__scroll-button--BackgroundColor);
+  border: initial;
```

Comment:
Is there a reason you use `initial` over `0`?

Reply:
This was used in the tab example so I thought it was purposeful. I can make a note to update both.